core/loader/rss.ts

```diff
@@ -12,16 +17,30 @@ function parsePosts(text: TextResponse): OriginPost[] {
         item.querySelector('guid')?.textContent ??
         item.querySelector('link')?.textContent
     )
-    .map(item => ({
-      full: item.querySelector('description')?.textContent ?? undefined,
-      media: [],
-      originId:
-        item.querySelector('guid')?.textContent ??
-        item.querySelector('link')!.textContent!,
-      publishedAt: toTime(item.querySelector('pubDate')?.textContent),
-      title: item.querySelector('title')?.textContent ?? undefined,
-      url: item.querySelector('link')?.textContent ?? undefined
-    }))
+    .map(item => {
+      const description = item.querySelector('description')
+
+      const imageNodes = description?.querySelectorAll('img') ?? []
+      const descriptionImages = [...imageNodes].reduce(
+        getImgReducer('src'),
+        new Set()
+      )
+
+      const allImages = [
+        ...item.querySelectorAll('media\\:content[medium=image]')
```

Comment:
Am I right that developer can give any name to XML namespace? It could be not just `media:` but also any `mrss:` and anything else.

I think DOM has an API to use namespace URL for these cases.